usecase/vehicle.go

```diff
@@ -0,0 +1,24 @@
+package usecase
+
+import (
+	"github.com/edgarduran/Go-Dispatch-Bootcamp/models"
+	"github.com/edgarduran/Go-Dispatch-Bootcamp/service"
+)
+
+func GetAllVehicles() ([]models.Vehicle, error) {
+	vehicles, err := service.GetAllVehicles()
+	if err != nil {
+		return nil, err
```

Comment:
Errors obtained from an external package's method call should be wrapped in order to better describe the reason for the error that is being returned.

Please, fix this for this and all other occurrences of this issue.

```suggestion
		return nil, fmt.Errorf("getting all vehicles: %v", err)
```